### PR TITLE
Fix #158 - convert step inputs column to mediumtext on MySQL/MariaDB

### LINKED ISSUE #158: Support for huge inputs with mysql
Seems like dynflow_actions.data column is a TEXT type which on MySQL limits the input to 64kB. For large inputs (e.g. foreman_chef attributes import) it's not enough. I'd suggest using MEDIUMTEXT, 16MB *ought to be enough for anybody*

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Sequel.migration do
+  up do
+    alter_table(:dynflow_steps) do
+      if Sequel::Model.db.database_type == :mysql
```
Comment (iNecas) [inline lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb]: using `@db` might be a bit safer here, as I'm not sure how it is with the global attribute `Sequel::Model.db` when there would be more sequel instances connected to different adapters.

Comment (ares) [inline lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb]: How do I set this `@db`? It does not seem to be present in migration block. I suppose I should ask world instance but I'm not sure how to get it in migration.

**Comment by adamruzicka** [comment]
`@db` should be accessible and already set to the db being migrated inside the `alter_table` blocks

File: lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb (hunk 1 above)
Comment (adamruzicka) [inline lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb]: @db should be accessible and already set to the db being migrated inside the alter_table blocks

File: lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        set_column_type :data, :mediumtext
+      end
+    end
+  end
+
+  down do
+    if Sequel::Model.db.database_type == :mysql
+      set_column_type :data, :text
```
Comment (adamruzicka) [inline lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb]: Wouldn't this need to be wrapped in an `alter_table` block or given a table symbol directly?

**Comment by ares** [comment]
ok fixed both, thanks

**Comment by iNecas** [comment]
Thanks @ares will merge after travis finishes

File: lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  up do
+    alter_table(:dynflow_steps) do
+      if @db.database_type == :mysql
+        set_column_type :data, :mediumtext
+      end
+    end
+  end
+
+  down do
+    alter_table(:dynflow_steps)
```
Comment (adamruzicka) [inline lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb]: missing `do` here

Comment (ares) [inline lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb]: Gah, that happens when you're in hurry. Sorry will fix it tomorrow

**Comment by ares** [comment]
[test]

**Comment by iNecas** [comment]
Thank you @ares

**Pull request merged**